src/test/java/uk/gov/hmcts/dts/fact/model/CourtReferenceTest.java

```diff
@@ -14,7 +14,7 @@
 
 class CourtReferenceTest {
     static Court courtEntity;
-    private static Timestamp currentTime = new Timestamp(System.currentTimeMillis());
+    private static final Timestamp currentTime = new Timestamp(System.currentTimeMillis());
```

Comment:
A very minor thing, the naming convention for "private static final"  in java is normal in upper case. So CURRENT_TIME instead of currentTime. I am not too fussed about that. Up to u to decide.